docs/3-opt-edit-experiment.rst

```diff
@@ -0,0 +1,51 @@
+==============
```

Comment:
This now seems covered by `add-a-benchmark`, `add-a-system-config`, and `add-an-experiment` (those sections cover this layout and explain the files in more detail).